### ISSUE TITLE
Support for multiple solvers

### ISSUE THREAD

**Issue by rubenvanparys** [meco-group/omg-tools#36]
For now omg-tools always used ipopt. Other solvers should be possible by option.

**Comment by rubenvanparys** [comment]
Syntax proposal:
`options['solver'] = 'ipopt' `
`options['solver_options'] = {'ipopt': {'ipopt.tol': 1e-4,...}}

**Comment by timmercy** [comment]
Maybe we should also add some examples with corresponding options (e.g. warm starting) + syntax?

**Comment by rubenvanparys** [comment]
@timmercy did you try already to interface snopt or wohrp from omg-tools?

**Comment by timmercy** [comment]
I added an example to the develop branch (p2p_holonomic_solvertest.py). 

I needed to add a line in problem.py in set_options(self, options), since otherwise self.options['solver_options'] = always ipopt

There are some open issues: 
  - worhp.qp_ipLsMethod is not recognized as a worhp setting --> probably syntax changed from CasADi 2.4 (where this works) to 3.0
  - snopt is not recognized as solver --> maybe also linked to CasADi 3.0
  - Problem tests on 'solve_succeeded', but this is ipopt-specific syntax --> how do we solve this?

Joris told me he will help with the first two issues later.
